src/panels/config/automation/condition/types/ha-automation-condition-time.ts

```diff
@@ -36,10 +61,71 @@ export class HaTimeCondition extends LitElement implements ConditionElement {
         .value=${before}
         @value-changed=${this._valueChanged}
       ></paper-input>
+
+      ${Object.keys(DAYS).map(
+        (day) => html`
+          <ha-formfield
+            alignEnd
+            spaceBetween
+            class="flex"
```

Comment:
Let's changed the class to describe what the element is not what the css should be. Flex could mean a hundred different things with display: flex

```suggestion
            class="weekday-toggles"
```

or something

Reply:
ok fixed